src/main/scala/Convertor.scala

```diff
@@ -0,0 +1,33 @@
+/**
+  * Created by Profile on 19/07/2017.
+  */
+class Convertor {
+
+  val englishConversion= List("thousand","million", "billion", "trillion", "quadrillion", "quintillion", "sextillion")
+  val frenchConversion= List("thousand", "million", "millard", "billion", "billiard", "trillion", "trilliard")
+
+
+
+
+  def convertNumber(number: String, conversion: String): String = {
+
+    def num(formattedNumber: String, typeOfConversion: List[String]): String = formattedNumber.length match {
+      case check if (formattedNumber.length <= 3) => formattedNumber
+      case 4 => formattedNumber.head.toString + " " + typeOfConversion(0) + " " + num(formattedNumber.tail, typeOfConversion)
```

Comment:
formattedNumber.head.toString this is called like 10 times, could you make into a helper method